src/homeork_2/Main.java

```diff
@@ -0,0 +1,97 @@
+package com.second;
+
+import com.second.Model.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+
+public class Main {
+
+    public static void main(String[] args) {
+        ArrayList<Student> students = new ArrayList<>();
+        students.add(new Student("Anna", "Danielyan", 20));
+        students.add(new Student("Nelli", "Karapetuyan", 19));
+        students.add(new Student("Arman", "Tovmassyan", 18));
+        students.add(new Student("Roland", "Barseghyan", 16));
+
+        Collections.sort(students, new StudentComparatorUsingFirstName());
+        System.out.println("Sorting students by firstName in ascending order: ");
+        printStudents(students);
+
+        System.out.println();
+
+        Collections.sort(students, new StudentComparatorUsingAge());
+        System.out.println("Sorting students by age in descending order: ");
+        printStudents(students);
+
+        System.out.println();
+        Collections.sort(students, Student::compareTo);
```

Comment:
Lusine jan homwork was to sort list of students by firstname and age(desc) at the same time and sort LinkedList by only lastname

Reply:
btw, if you just use  Collections.sort(students); it will sort them by Student's comparator